### PR TITLE
L1t rcttodigi

### PR DESCRIPTION
**Pull request by mulhearn**
This is a reboot of PR #9931.  This is an initial commit for the RCT unpacker using the Stage-1 Upgrade Readout using Fed 1350.

### CODE REVIEW

**Comment by mulhearn** [comment]
please test

**Comment by mulhearn** [comment]
@isobelojalvo, this is of interest to you.

**Comment by isobelojalvo** [comment]
@mulhearn  Thank you!!

**Comment by isobelojalvo** [comment]
Awesome. :)

**Comment by mulhearn** [comment]
+1

File: EventFilter/RctRawToDigi/plugins/RctRawToDigi.cc [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+using edm::LogError;
+
+
+RctRawToDigi::RctRawToDigi(const edm::ParameterSet& iConfig) :
+  inputLabel_(iConfig.getParameter<edm::InputTag>("inputLabel")),
+  fedId_(iConfig.getUntrackedParameter<int>("rctFedId", FEDNumbering::MINTriggerUpgradeFEDID)),
+  verbose_(iConfig.getUntrackedParameter<bool>("verbose",false))
+{
+  LogDebug("RCT") << "RctRawToDigi will unpack FED Id " << fedId_;
+  std::cout<< "RctRawToDigi will unpack FED Id " << fedId_<<std::endl;
```
Comment (davidlange6) [inline EventFilter/RctRawToDigi/plugins/RctRawToDigi.cc]: @mulhearn - could you remove this cout. thanks

**Pull request merged**